.github/workflows/ipc_fuzzer.yml

```diff
@@ -10,39 +10,6 @@ on: [pull_request, workflow_dispatch]
 
 jobs:
 
-
-  # For the actual fuzzer see tools/oss-fuzz/README.
-  # also see
-  # https://google.github.io/oss-fuzz/getting-started/continuous-integration/
-  #
-  # Build and run fuzzer for 5s just to check that it runs properly. If it
```

Comment:
is this line still not true?

Reply:
It is, but it's "3m" now and done with the newer rig

Reply:
Why 3m? We just want to smoke test in this CI and let OSS-fuzz handle the longer runs

Reply:
> I guess my gut says that fuzzing is a technique for validation of new code. It has minimal (but sure, not zero) value on maintenance branches that are expected to be protocol-compatible in perpetuity.

Exactly my thoughts, and its isolated to finding holes in communication paths. I think a better option would be to mandate fixes for IPC3 go back to that branch if they apply rather than trying to fuzz it directly.